docs/prover/Report/CallResolutionTable.md

```diff
@@ -0,0 +1,56 @@
+CallResolutionTable
+=================
+
+The `CallResolutionTable` shows information about all the summarized calls in the program.
+It helps the user to better understand decisions made by the Prover- which calls in the rule code got inlined, which were replaced by summaries, and why.
+
+Attributes
+------
+
+ * **Caller**- The caller of the call. Always resolved by the Prover.
+
+ * **Callee**- The callee of the call. For internal calls, always resolved. For external calls, might be unresolved, depending on linking, summarization, and more.
+See below.
+
+ * **Call Site**- source information of the call itself. Shows the location of the call (which file, which line), and a snippet from the source code (the invocation itself).
+
+ * **Summary**- what summary got applied for the call.
+
+ * **Comments**- a list of comments about the resolution of the callee (which as mentioned above, might be unresolved), the applied summarization.
```

Comment:
* **Comments**- a list of comments about the resolution of the callee. (As mentioned above, this might be unresolved).